config/routes.rb

```diff
@@ -199,7 +199,7 @@
   get 'projects/:project_id/wiki/:id/toc' => 'wiki#index', as: 'wiki_page_toc'
   post 'projects/:project_id/wiki/preview' => 'wiki#preview', as: 'preview_wiki'
   post 'projects/:id/wiki' => 'wikis#edit'
-  match 'projects/:id/wiki/destroy' => 'wikis#destroy'
+  delete 'projects/:id/wiki/destroy' => 'wikis#destroy'
```

Comment:
@ulferts 

> And while you are at it, there are other verify statements in that controller as well.

there are a lot of `verify` statements throughout the application. I'd obviously like to get rid of them completely, letting the router verify the correct HTTP verb. I think the right time for doing that though is when we eliminate the default routes completely ([Bug 4996](https://community.openproject.org/work_packages/4996)).

With this PR, we still have a default route for `GET` requests:

    get '/:controller(/:action(/:id))'

To enable this to happen, routing specs should be cleaned up first, including splitting up `spec/legacy/integration/routing_spec.rb`.